projects/ngrx.io/content/marketing/contributing.md

```diff
@@ -0,0 +1,224 @@
+- `src\index.ts` - This file is responsible for providing the public API surface layer. Only public API methods and models are exported from here.
+
+### Setup
+
+To ensure the correct project dependencies are installed, please run the following command before each new pull request (PR) is submitted. This command should be run from the root project folder.
+
+```sh
+yarn
+```
+
+### Build
+
+NgRx uses `Bazel` for builds. In order to build locally, please run the following command:
+
+```sh
+yarn build
+```
+
+### Testing
+
+It is good practice to run the following command from the root project folder before pushing new commits.
+
+```sh
+yarn test
+```
+
+## Submitting Pull Requests
+
+**Please follow these basic steps to simplify pull request reviews. If you don't you'll probably just be asked to anyway.**
+
+- Please rebase your branch against the current master.
+- Run the `Setup` command to make sure your development dependencies are up-to-date.
+- Please ensure the test suite passes before submitting a PR.
+- If you've added new functionality, **please** include tests which validate its behavior.
+- Make reference to possible [issues](https://github.com/ngrx/platform/issues) on PR comment.
+
+## Submitting bug reports
+
+- Search through issues to see if a previous issue has already been reported and/or fixed.
+- Provide a _small_ reproduction using a [StackBlitz project](https://stackblitz.com/edit/ngrx-seed) or a GitHub repository.
+- Please detail the affected browser(s) and operating system(s).
+- Please be sure to state which version of Angular, node and npm you're using.
+
+## Submitting new features
+
+- We value keeping the API surface small and concise, which factors into whether new features are accepted.
+- Submit an issue with the prefix `RFC:` with your feature request.
+- The feature will be discussed and considered.
+- Once the PR is submitted, it will be reviewed and merged once approved.
+
+## <a name="commit"></a> Commit Message Guidelines
+
+We have very precise rules over how our git commit messages can be formatted. This leads to **more
+readable messages** that are easy to follow when looking through the **project history**. But also,
+we use the git commit messages to **generate the NgRx change log**.
+
+### Commit Message Format
+
+Each commit message consists of a **header**, a **body** and a **footer**. The header has a special
+format that includes a **type**, a **scope** and a **subject**:
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+The **header** is mandatory and the **scope** of the header is optional.
+
+Any line of the commit message cannot be longer 100 characters! This allows the message to be easier
+to read on GitHub as well as in various git tools.
+
+The footer should contain a [closing reference to an issue](https://help.github.com/articles/closing-issues-via-commit-messages/) if any.
+

```

Comment:
```suggestion
```
feat(router-store): add routerState config option

Closes #1834
```
```

Reply:
I believe I have updated this one as requested, can you confirm?

Reply:
👍